charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -325,6 +362,8 @@ data:
           target: https://github.com/parodos-dev/workflow-software-templates/blob/main/entities/workflow-resources.yaml
```

Comment:
Also, what do you think if we use a dedicated branch for these manifests in 1.2? This way allows us to continue making changes in main and backport cherry picked changes to 1.2 without disrupting any production usage of these manifests.